src/components/BorrowerProfile/LendCta.vue

```diff
@@ -417,9 +421,9 @@ export default {
 			type: Boolean,
 			default: false,
 		},
-		enableActivityFeed: {
-			type: Boolean,
-			default: false,
+		activities: {
+			type: Object,
+			default: () => ({}),
```

Comment:
If we are checking on whether this is defined, then this default should also be `null`